samples/unit-tests/polar/column-inverted/demo.js

```diff
@@ -0,0 +1,204 @@
+QUnit.skip('Positions of the points.', function (assert) {
```

Comment:
Fix these tests for the refactored code (`startR` no longer exists on point).

Reply:
Instead of using the `startR`, the `start` and `end` are compared in order to check if points are outside the visible range.